tests/e2e/specs/in-game/basicMoves.spec.js

```diff
@@ -373,7 +373,7 @@ describe('Game Basic Moves - P0 Perspective', () => {
     // Should have 3 move options
     cy.get('[data-move-choice]').should('have.length', 3);
     cy.get('[data-cy=cancel-move]').click();
-    cy.get('#move-choice-overlay').should('not.be.visible');
+    cy.get('#move-choice-overlay').should('not.exist');
```

Comment:
When the overlay was closed, `#move-choice-overlay` was still expected to be found, just not visible, but with the requirement change, it's not present at all. 

This was the only instance of `cy.get('#move-choice-overlay').should('not.be.visible');` in the whole testing suite.